apps/api/lib/email/daily-cronjob-reminder-email-templates/mentorship-is-weeks-old-so-request-feedback_mentee.mjml

```diff
@@ -14,20 +14,20 @@
           <mj-text mj-class="text" padding="0 0 20px 0"
             >Dear ${menteeFirstName},</mj-text
           >
-          <mj-text mj-class="text" padding="0 0 20px 0"
+          <mj-text mj-class="text paragraph" 
```

Comment:
Wow, big improvement to get rid of all those repeated `padding="0 0 20px 0"` 😅 This works fine?

Reply:
Yes, it works nicely!

Reply:
Sweet.